applications/FemToDemApplication/python_scripts/MainCouplingPfemFemDem.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import print_function, absolute_import, division  #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as KM
+import KratosMultiphysics.FemToDemApplication as FEMDEM
+import KratosMultiphysics.PfemFluidDynamicsApplication as PFEM
+import KratosMultiphysics.FemToDemApplication.MainCouplingFemDem    as MainCouplingFemDem
+import KratosMultiphysics.FemToDemApplication.MainPFEM_for_coupling as MainPFEM_for_coupling
+
+def Wait():
+    input("PFEM-FEMDEM -> Press Something")
+
+def KratosPrintInfo(message):
+    """This function prints info on screen
+    """
+    KM.Logger.Print(message, label="")
+    KM.Logger.Flush()
+#============================================================================================================================
+class MainCouplingPfemFemDem_Solution:
+#============================================================================================================================
+
+    def __init__(self, Model, PFEMparameters):
+        # Initialize solutions of the FEMDEM and PFEM
+        self.FEMDEM_Solution = MainCouplingFemDem.MainCoupledFemDem_Solution(Model)
+        self.FEMDEM_Solution.Initialize()
+
+        self.PFEM_Solution = MainPFEM_for_coupling.MainPFEM_for_coupling_solution(Model, 
+                                                                                  self.FEMDEM_Solution.FEM_Solution.main_model_part,
+                                                                                  PFEMparameters)
+        KratosPrintInfo("   ___                  _          _            _ _   _         ___  ___  __        ")
+        KratosPrintInfo("  / __\___  _   _ _ __ | | ___  __| | __      _(_) |_| |__     / _ \/ __\/__\/\/\   ")
+        KratosPrintInfo(" / /  / _ \| | | | '_ \| |/ _ \/ _` | \ \ /\ / / | __| '_ \   / /_)/ _\ /_\ /    \  ")
+        KratosPrintInfo("/ /__| (_) | |_| | |_) | |  __/ (_| |  \ V  V /| | |_| | | | / ___/ /  //__/ /\/\ \ ")
+        KratosPrintInfo("\____/\___/ \__,_| .__/|_|\___|\__,_|   \_/\_/ |_|\__|_| |_| \/   \/   \__/\/    \/ ")
+        KratosPrintInfo("")
```

Comment:
You can just simply add \n at the end o f the last one.

BTW: I would join one log, and remember that the first argument of the logger is the label (you are printing the label)

Reply:
OK! changing. THX